resources/helper/PerlinHelper.gd

```diff
@@ -0,0 +1,17 @@
+extends Node
```

Comment:
Why Node?

Reply:
Bc godot is dumb sometimes and defaulted to that again on my PC